RULES.md

```diff
@@ -301,3 +303,22 @@ True
 True
 ```
 
+<a name="String" />
+
+#### String
+
+The field under validation must be a String
+```python
+>>> from validator import validate
+
+>>> reqs = {'value' : 'some string'}
```

Comment:
Sorry missed this one. Please rewrite this in the String rule docstring to use double quotes rather than one and then re run `make readme`

Reply:
Sure thing